server/utils/risksAndNeeds/attitudesUtils.ts

```diff
@@ -0,0 +1,18 @@
+import ShowRisksAndNeedsUtils from '../referrals/showRisksAndNeedsUtils'
+import type { Attitude } from '@accredited-programmes/models'
+import type { GovukFrontendSummaryListRowWithKeyAndValue } from '@accredited-programmes/ui'
+
+export default class AttitudesUtils {
```

Comment:
I notice the entity and endpoint use singular `Attitude`, but the section is `Attitudes`. I think the division in this code is type/client/service use singular, the rest plural? It's not something we can fully control on the UI, but it might be good to note the discrepancy somewhere and explain the motivation for the specific point at which we switch to plural (entity vs presentation)?